miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/LabRestControllerST.java

```diff
@@ -49,6 +48,7 @@
 public class LabRestControllerST extends AbstractST {
 
   private static final String CONTROLLER_BASE = "/rest/labs";
+  private static final Class<LabImpl> controllerClass = LabImpl.class;
 
   @Test
   public void testBulkCreateAsync() throws Exception {
```

Comment:
It's probably valuable to add a comment here that there are no restrictions on who can create a lab. (And makes it clearer why this class doesn't have a `testBulkCreateAsyncFail` test, unlike say the SopRestControllerST).